sdk/eventhub/event-hubs/consuming-events.md

```diff
@@ -0,0 +1,85 @@
+# Consuming events
+
+The `EventHubConsumerClient` provides a _push-based_ model for receiving events.
```

Comment:
Can we skip the reference to "push based model"? How about

> The `EventHubConsumerClient` lets you subscribe to events from one or all partitions of an Event Hub.
You do this by invoking the `subscribe()` method on the client which returns a subscription. Calling `close()`
on the subscription stops reading of any more events.

I would suggest skipping the 3 bullet points talking about what you can pass to the `subscribe()` method as that can be inferred by looking at the method signature.

Then jump to line 13 where you talk about what happens for **each** partition that is being read

Reply:
Updated!